### PR TITLE
Added README for candidate list module

### PR DESCRIPTION
**Pull request by driusan**
This adds a README for the candidate list module, following
the template of the imaging browser.

### CODE REVIEW

File: modules/candidate_list/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+## Purpose
+
+The candidate list is intended to provide a menu from which users
+can access specific candidates while browsing LORIS data.
+
+## Intended Users
+
+The candidate list is primarily used by data entry staff to use in
+order to access the `timepoint_list` module (and `instrument_list`
+from there) in order to access the candidate and perform data entry
```
Comment (PapillonMcGill) [inline modules/candidate_list/README.md]: missing end point (.)

File: modules/candidate_list/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+candidate in LORIS, while users with only the `data_entry` permission
+can only see candidates at study sites where they are affiliated.
+
+## Configurations
+
+The "useProjects" configuration affects whether the table has a
+"Project" column for the candidate, and whether there is a filter
+dropdown for that column.
+
+The "useEDC" configuration variable has a similar function for the
```
Comment (PapillonMcGill) [inline modules/candidate_list/README.md]: should back tick (`) be used instead of double quote (") for useEDC?

useProjects (line 31) is similar

File: modules/candidate_list/README.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+EDC column.
```
Comment (PapillonMcGill) [inline modules/candidate_list/README.md]: should  EDC be enclosed with double quote?

File: modules/candidate_list/README.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+## Interactions with LORIS
+
+The "Scan Done" column indicates whether or not a scan was performed
+for that candidate. When a scan was performed (ie it has the value
```
Comment (PapillonMcGill) [inline modules/candidate_list/README.md]: ...(**i.e.,** it ...

File: modules/candidate_list/README.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+prefiltered for that candidate.
+
+Clicking on the cell in the PSCID column takes the user to the
+`timepoint_list` page for that candidate. The link is only clickable
+if the user does *not* have the `data_entry` permission. For users
+who have the `data_entry` permission, a separate form appears for
+the user to type the CandID/PSCID combo in order to access the
+candidate.  This is intended to prevent data entry errors caused
+by accidentally clicking on the wrong row before doing data entry.
+
```
Comment (PapillonMcGill) [inline modules/candidate_list/README.md]: empty line, should be removed

File: modules/candidate_list/README.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+"Project" column for the candidate, and whether there is a filter
+dropdown for that column.
+
+The `useEDC` configuration variable has a similar function for the
+"EDC" column.
+
+## Interactions with LORIS
+
+The "Scan Done" column indicates whether or not a scan was performed
+for that candidate. When a scan was performed (i.e. it has the value
```
Comment (PapillonMcGill) [inline modules/candidate_list/README.md]: need a coma after i.e.

Comment (driusan) [reply modules/candidate_list/README.md]: No it doesn't.

Comment (PapillonMcGill) [reply modules/candidate_list/README.md]: I have check a few grammar web site and they all state that a comma is needed or at least put one after

File: modules/candidate_list/README.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
+# Candidate List
+
+## Purpose
+
+The candidate list is intended to provide a menu from which users
+can access specific candidates while browsing LORIS data.
```
Comment (christinerogers) [inline modules/candidate_list/README.md]: The most important piece of context is how this module is used/known:  
e.g. I'd add:
"This module is known to users in the front end as "Access Profiles" and is the most heavily accessed module in LORIS.   

It provides an at-a-glance view of data available across all candidates stored in LORIS."

File: modules/candidate_list/README.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+
+## Intended Users
+
+The candidate list is primarily used by data entry staff to use in
+order to access the `timepoint_list` module (and `instrument_list`
+from there) in order to access the candidate and perform data entry.
+
```
Comment (christinerogers) [inline modules/candidate_list/README.md]: Across all user groups, this module is used for filtering and candidate profiles and reviewing candidate data status.

File: modules/candidate_list/README.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+## Scope
+
+The `candidate_list` module provides a list of candidates, as well
+as linking to other relevant LORIS modules where applicable.
+
```
Comment (christinerogers) [inline modules/candidate_list/README.md]: If any imaging data or outstanding behavioural feedback comments exist, this will be visible for a given candidate.

File: modules/candidate_list/README.md [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+can only see candidates at study sites where they are affiliated.
+
+## Configurations
+
+The `useProjects` configuration affects whether the table has a
+"Project" column for the candidate, and whether there is a filter
+dropdown for that column.
+
+The `useEDC` configuration variable has a similar function for the
+"EDC" column.
```
Comment (christinerogers) [inline modules/candidate_list/README.md]: Because nobody knows what EDC is, I'd add: 
"And is used for studies where prenatal visits are registered (EDC = estimated date of confinement or expected birthdate)"

File: modules/candidate_list/README.md [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+
+The "Scan Done" column indicates whether or not a scan was performed
+for that candidate. When a scan was performed (i.e. it has the value
+"Y"), clicking the table cell links to the `imaging_browser` module
+prefiltered for that candidate.
+
+Clicking on the cell in the PSCID column takes the user to the
+`timepoint_list` page for that candidate. The link is only clickable
+if the user does *not* have the `data_entry` permission. For users
+who have the `data_entry` permission, a separate form appears for
```
Comment (christinerogers) [inline modules/candidate_list/README.md]: is this behaviour 100% triple-checked?   I trust you if so, but I'm not confident it behaves this way.

File: modules/candidate_list/README.md [hunk 12]
```diff
@@ -0,0 +1,51 @@
...
+
+The `useProjects` configuration affects whether the table has a
+"Project" column for the candidate, and whether there is a filter
+dropdown for that column.
+
+The `useEDC` configuration variable has a similar function for the
+"EDC" column.
+
+## Interactions with LORIS
+
```
Comment (christinerogers) [inline modules/candidate_list/README.md]: Dare we mention Bvl feedback? :)

File: modules/candidate_list/README.md (hunk 6 above)
Comment (christinerogers) [reply modules/candidate_list/README.md]: (fwiw, you're both right.  There's definitely a rule about "i.e.," having a comma, and it's often ignored in informal and technical writing, which is fine by me.)

File: modules/candidate_list/README.md [hunk 13]
```diff
@@ -0,0 +1,51 @@
+# Candidate List
+
+## Purpose
+
+The candidate list is intended to provide a menu from which users
+can access specific candidates while browsing LORIS data.
+
+## Intended Users
+
+The candidate list is primarily used by data entry staff to use in
```
Comment (cmadjar) [inline modules/candidate_list/README.md]: "used" quickly followed by "to use". Maybe rephrase to keep only one instance of the word use?

File: modules/candidate_list/README.md [hunk 14]
```diff
@@ -0,0 +1,51 @@
...
+order to access the `timepoint_list` module (and `instrument_list`
+from there) in order to access the candidate and perform data entry.
+
+## Scope
+
+The `candidate_list` module provides a list of candidates, as well
+as linking to other relevant LORIS modules where applicable.
```
Comment (cmadjar) [inline modules/candidate_list/README.md]: Shouldn't this be "links" instead of "linking"?

File: modules/candidate_list/README.md [hunk 15]
```diff
@@ -0,0 +1,51 @@
...
+
+## Permissions
+
+Accessing the `candidate_list` module requires either the `data_entry`
+permission and at least one study site, or the `access_all_profiles`
+permission.
+
+Users with the `access_all_profiles` permission can see every
+candidate in LORIS, while users with only the `data_entry` permission
+can only see candidates at study sites where they are affiliated.
```
Comment (cmadjar) [inline modules/candidate_list/README.md]: Replace "only see candidates" by "solely see candidates" or "exclusively see candidates" to avoid repetition of the word only so close to each other.

File: modules/candidate_list/README.md (hunk 11 above)
Comment (cmadjar) [reply modules/candidate_list/README.md]: @driusan maybe check with @Jkat as I don't recall this is the case for PREVENT-AD. Maybe it is configurable or PREVENT-AD is overwriting the module. Both are possible.

File: modules/candidate_list/README.md (hunk 10 above)
Comment (cmadjar) [reply modules/candidate_list/README.md]: I agree, it would be good to at least define what is EDC as not everyone knows about this.

File: modules/candidate_list/README.md (hunk 7 above)
Comment (xlecours) [reply modules/candidate_list/README.md]: `..is the most heavily accessed module in LORIS.` seems speculative

**Comment by ridz1208** [comment]
@driusan please make changes at your earliest convenience

File: modules/candidate_list/README.md (hunk 7 above)
Comment (johnsaigle) [reply modules/candidate_list/README.md]: I think the note about the front-end name is a good suggestion, though.

File: modules/candidate_list/README.md [hunk 16]
```diff
@@ -28,17 +28,17 @@ can only see candidates at study sites where they are affiliated.
...
 
-The "useProjects" configuration affects whether the table has a
+The `useProjects` configuration affects whether the table has a
 "Project" column for the candidate, and whether there is a filter
 dropdown for that column.
 
-The "useEDC" configuration variable has a similar function for the
-EDC column.
+The `useEDC` configuration variable has a similar function for the
+"EDC" column.
```
Comment (PapillonMcGill) [inline modules/candidate_list/README.md]: Shouldn't it be ` instead of " to be consistent?

File: modules/candidate_list/README.md (hunk 12 above)
Comment (PapillonMcGill) [reply modules/candidate_list/README.md]: @christinerogers  What's the meaning of Bvl?

File: modules/candidate_list/README.md (hunk 16 above)
Comment (paiva) [reply modules/candidate_list/README.md]: I think it is fine, ` is more for variable names.

File: modules/candidate_list/README.md [hunk 17]
```diff
@@ -0,0 +1,51 @@
...
+
+## Interactions with LORIS
+
+The "Scan Done" column indicates whether or not a scan was performed
+for that candidate. When a scan was performed (i.e. it has the value
+"Y"), clicking the table cell links to the `imaging_browser` module
+prefiltered for that candidate.
+
+Clicking on the cell in the PSCID column takes the user to the
```
Comment (paiva) [inline modules/candidate_list/README.md]: Should be "PSCID" to be consistent with column naming convention (like @PapillonMcGill suggested).

File: modules/candidate_list/README.md (hunk 7 above)
Comment (driusan) [reply modules/candidate_list/README.md]: I added something about the frontend.

File: modules/candidate_list/README.md (hunk 17 above)
Comment (driusan) [reply modules/candidate_list/README.md]: It's referred to as PSCID everywhere in LORIS.

File: modules/candidate_list/README.md (hunk 11 above)
Comment (driusan) [reply modules/candidate_list/README.md]: If this isn't the behaviour, it's a bug in the code that should be fixed.
